app/ide-desktop/lib/dashboard/src/authentication/src/index.tsx

```diff
@@ -43,8 +43,12 @@ export function run(props: app.AppProps) {
     if (root == null) {
         logger.error(`Could not find root element with ID '${ROOT_ELEMENT_ID}'.`)
     } else {
-        // This element is re-added by the `Ide` component.
-        document.getElementById(IDE_ELEMENT_ID)?.remove()
+        // FIXME[sb]: This is a temporary workaround and will be fixed
+        // when IDE support is properly integrated into the dashboard.
```

Comment:
Every fixme should:
1. Provide a clear semantic information of what it does. Here I see fixme and I don't understand what it really does and why it is considered a "hack".
2. Provide link to an issue where it will be fixed.

Reply:
This was just a rollback but provided with proper format.